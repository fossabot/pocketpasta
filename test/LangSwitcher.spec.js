import { shallowMount, createLocalVue } from '@vue/test-utils';
import Component from '@/components/Layout/LangSwitcher';
import Vuex from 'vuex';

const localVue = createLocalVue();

localVue.use(Vuex);

describe('LangSwitcher', () => {
  const factory = () => shallowMount(Component, {});

  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
