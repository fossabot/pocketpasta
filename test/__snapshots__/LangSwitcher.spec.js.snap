// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LangSwitcher renders properly 1`] = `"<v-menu> <v-list><v-list-tile><v-list-tile-title><nuxt-link to=\\"en\\">English</nuxt-link></v-list-tile-title></v-list-tile></v-list></v-menu>"`;
